binder/postBuild

```diff
@@ -14,12 +14,13 @@ SCALA_VERSION=2.12.8 ALMOND_VERSION=0.4.0
   -o almond
 ./almond --install \
   --command "java -XX:MaxRAMPercentage=80.0 -jar almond" \
-  --copy-launcher true
+  --copy-launcher true \
+  --metabrowse true
```

Comment:
Just `--metabrowse` is enough (or `--metabrowse=true`, but `--metabrowse true` shouldn't work)

Reply:
Hmm, should be the same for `--copy-launcher` actually.

Reply:
Ah thanks. Changed now.